cmd/export_effects.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/stellar/stellar-etl/internal/input"
+	"github.com/stellar/stellar-etl/internal/transform"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+var effectsCmd = &cobra.Command{
+	Use:   "export_effects",
+	Short: "Exports the effects data over a specified range",
+	Run: func(cmd *cobra.Command, args []string) {
+		cmdLogger.SetLevel(logrus.InfoLevel)
+		endNum, strictExport, isTest, extra := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		cmdLogger.StrictExport = strictExport
+		startNum, path, limit := utils.MustArchiveFlags(cmd.Flags(), cmdLogger)
+		gcsBucket, gcpCredentials := utils.MustGcsFlags(cmd.Flags(), cmdLogger)
+		env := utils.GetEnvironmentDetails(isTest)
+
+		operations, err := input.GetOperations(startNum, endNum, limit, env)
+		if err != nil {
+			cmdLogger.Fatal("could not read effects: ", err)
```

Comment:
operations, not effects?
```suggestion
			cmdLogger.Fatalf("could not read operations in [%d, %d] (limit=%d): %v", startNum, endNum, limit, err)
```